lib/main.js

```diff
@@ -225,10 +225,13 @@ function getUserCerts() {
   let certList = certCache.getX509CachedCerts();
   let userCerts = [];
   let certListEnumerator = certList.getEnumerator();
+  let now = Date.now();
   while (certListEnumerator.hasMoreElements()) {
     let cert = certListEnumerator.getNext().QueryInterface(Ci.nsIX509Cert);
     if (cert.certType & Ci.nsIX509Cert.USER_CERT) {
-      userCerts.push(cert);
+      if (now <= cert.validity.notAfter/1000) {
```

Comment:
A couple of comments:
1. In terms of style, I've been using spaces between operators: 'notAfter / 1000'
2. Shouldn't we also check that now >= cert.validity.notBefore / 1000?
3. It would be nice to have a comment along the lines of "// notAfter and notBefore are microseconds since the epoch, whereas now is in milliseconds"